### PR TITLE
Feature/delete user

### PR DESCRIPTION
**Pull request by abdessalamboulayat**

### CODE REVIEW

File: app/routes/user.py [hunk 1]
```diff
@@ -3,4 +3,22 @@
...
+userRouter: APIRouter = APIRouter(prefix="/user")
+
+@userRouter.patch("/{user_id}", status_code=205)
+async def modify_user(user_id: int, user_model: PydanticUserModify) -> None:
+    """
+    This controllers is used when modifying user informations.
+    """
+    await UserService.modify_user(user_id, user_model)
+
+@userRouter.delete("/{user_id}", status_code=204)
```
Comment (Maximelego) [inline app/routes/user.py:19]: Wrong error code used. Should be 205.
https://developer.mozilla.org/fr/docs/Web/HTTP/Status/205

File: app/services/UserService.py [hunk 2]
```diff
@@ -31,3 +31,15 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     except ValueError as e:
         raise HTTPException(status_code=409, detail=str(e)) from e
+
+async def delete_user(user_id: int):
```
Comment (Maximelego) [inline app/services/UserService.py:35]: Very good ! 
Typing is good, documentation was added, very nice !

**Review by Maximelego — CHANGES_REQUESTED** [review]
Only the response code should be modified.
However, great work !

File: app/routes/user.py (hunk 1 above)
Comment (abdessalamboulayat) [reply app/routes/user.py]: Done

Comment (Maximelego) [reply app/routes/user.py]: You can click the "resolve conversation" button when the subject is closed.

**Review by Maximelego — APPROVED** [review]
LGTM

**Review by HugoBien — APPROVED** [review]
LGTM

**Pull request merged**